template/.github/workflows/ci.yml

```diff
@@ -53,6 +53,6 @@ jobs:
     uses: ./.github/workflows/docs.yml
     with:
       publish: false
-      linkcheck: ${{ contains(matrix.variant.os, 'ubuntu') && github.ref == 'refs/heads/main' }}
+      linkcheck: github.ref == 'refs/heads/main' }}
```

Comment:
This was broken. The docs job does not have a matrix. This means that linkcheck never run in our current projects.

Reply:
That's interesting!